src/main/java/org/jabref/gui/Base.css

```diff
@@ -2431,3 +2431,16 @@ journalInfo .grid-cell-b {
 #styleSelectDialog .currentStyleNameLabel {
     -fx-font-size: 1em; -fx-font-weight: bold; -fx-text-fill: -jr-theme;
 }
+
+.dynamic-tooltip {
+    -fx-max-width: 70vw;
+    -fx-max-height: 50vh;
+    -fx-background-color: #B0C4DE;
+    -fx-font-size: 12px;
+    -fx-wrap-text: true;
+    -fx-padding: 8px;
+    -fx-background-radius: 6px;
+    -fx-border-color: #B0B8D9;
+    -fx-border-width: 0.8px;
+    -fx-effect: dropshadow(gaussian, rgba(0, 0, 0, 0.15), 4, 0, 0, 2);
+}
```

Comment:
vw and vw are not supported by javafx css.
See https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#typesize
Use em or ex for responsive design.
Do not alter fonz size.
Use standard colors defined in base.css instead of custom ones (defined by hexval here).
Please use three-pass-box for dropshadow and extract that as a new shadow color definition in base.css. Please unify the use of a dropshadow in the other places it is used as an effect.